funding/manager.go

```diff
@@ -2943,18 +2934,9 @@ func (f *Manager) receivedFundingLocked(node *btcec.PublicKey,
 	default:
 	}
 
-	// Check whether the peer is online. If it's not, we'll wait for them
-	// to come online before proceeding. This is to avoid a tight loop if
-	// they're offline.
-	connected := make(chan lnpeer.Peer, 1)
-	var peerKey [33]byte
-	copy(peerKey[:], node.SerializeCompressed())
-	f.cfg.NotifyWhenOnline(peerKey, connected)
-
-	select {
-	case <-connected:
-	case <-f.quit:
-		return false, ErrFundingManagerShuttingDown
+	// Avoid a tight loop if peer is offline.
```

Comment:
nit: perhaps we should preserve a bit more of the original comment. Something like: `Wait for the peer to come online before proceeding in order to avoid a tight look if they are offline`

Reply:
The original comment seems like overkill.  The "Check if online, if not wait for them to come online" part is clear from the self-documenting function name `waitForPeerOnline`.